release_notes/release_notes.stex

```diff
@@ -2726,6 +2726,14 @@ in fasl files does not generally make sense.
 %-----------------------------------------------------------------------------
 \section{Bug Fixes}\label{section:bugfixes}
 
+\subsection{Random number generation for large exact integers (10.1.0)}
+
+For an argument exact integer greater than 4294967087,
```

Comment:
For an argument exact integer => Given an exact integer

Reply:
Typos fixed — thanks!